### PR TITLE
Add namespace, update compileSdkVersion

### PR DESCRIPTION
**Pull request by vahellame**

### CODE REVIEW

File: packages/flutter_nekoton_bridge/android/build.gradle [hunk 1]
```diff
@@ -58,7 +58,8 @@ apply plugin: 'com.android.library'
 //     }
 // }
 android {
-    compileSdkVersion 31
+    namespace 'com.broxus.flutter_nekoton_bridge.flutter_nekoton_bridge'
```
Comment (nesquikm) [inline packages/flutter_nekoton_bridge/android/build.gradle:61]: Why you want to add namespace?

**Comment by nesquikm** [comment]
Please use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0) guidelines when creating PR (i.e. specify commit type fix/feat/etc).

**Comment by nesquikm** [comment]
@vahellame ping

File: packages/flutter_nekoton_bridge/android/build.gradle (hunk 1 above)
Comment (vahellame) [reply packages/flutter_nekoton_bridge/android/build.gradle:61]: Because with gradle 8.x.x and AGP 8.x.x namespace is required

Comment (nesquikm) [reply packages/flutter_nekoton_bridge/android/build.gradle:61]: Yeah, thank you!

File: packages/flutter_nekoton_bridge/android/build.gradle [hunk 2]
```diff
@@ -58,7 +58,8 @@ apply plugin: 'com.android.library'
...
+    compileSdkVersion 33
```
Comment (nesquikm) [inline packages/flutter_nekoton_bridge/android/build.gradle:62]: IMHO minCompileSdk is the right choice for a library.

Comment (nesquikm) [reply packages/flutter_nekoton_bridge/android/build.gradle:62]: https://developer.android.com/build/publish-library/prep-lib-release

**Pull request closed without merging**